lib/importer/sheets.js

```diff
@@ -46,7 +46,7 @@ exports.GetRows = (session, start=0, count=10) => {
 // Uses the session provided, which must contain a sheet name and a
 // mapping to perform the mapping of the data across the remaining
 // rows in the sheet to return an array of objects.
-exports.MapData = (session) => {
+exports.MapData = (session, previewLimit) => {
```

Comment:
We can specify the default here I think as we can say `(session, previewLimit=DEFAULT_PREVIEW_LIMIT`